odbc/src/handles/definitions.rs

```diff
@@ -200,6 +205,7 @@ impl Connection {
 pub struct Statement {
     pub connection: *mut MongoHandle,
     pub mongo_statement: Option<Box<dyn mongo_odbc_core::MongoStatement>>,
+    pub var_data_indicies: Option<HashMap<USmallInt, (usize, usize)>>,
```

Comment:
nit: spelling indicies -> indices.  
```suggestion
    pub var_data_indices: Option<HashMap<USmallInt, (usize, usize)>>,
```